src/helpers/single-value-providers.ts

```
@@ -0,0 +1,56 @@
+            getValue: () => Promise.resolve(Math.round(Math.random() * maxValue))
+        };
+    },
+    createCustomFieldIntegerValueProvider: (customField: string): CustomFieldIntegerValueProvider => {
+        return {
+            type: 'custom field integer',
+            customField: customField,
+            getValue: async () => {
+                if (!customField) return undefined;
+
+                const participantInfo = await MyDataHelps.getParticipantInfo();
+                if (participantInfo.customFields.hasOwnProperty(customField)) {
```

Biome

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)